client/src/pages/BasePage.jsx

```diff
@@ -12,11 +13,14 @@ import NavBar from "../components/Navbar";
 const BasePage = (props) => {
   const theme = useTheme();
 
+  const mockData = "02/14/2003";
+
   return (
     <Box sx={{ bgColor: theme.palette.background.default, height: "100vh" }}>
       <Container>
-        <NavBar />
+        {/* <NavBar /> */}
         {props.children}
+        <PostedOn date={mockData} />
```

Comment:
This might not be the component to include this component, if for testing purposes, we shouldn't commit this. Merging this would affect every page extending the `BasePage` component